### DISCUSSION TITLE
tutorials/intro-typescript/12-using-mappers

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#111] [course feedback]
# tutorials/intro-typescript/12-using-mappers

https://www.apollographql.com/tutorials/intro-typescript/12-using-mappers

**Comment by pr1thu** [comment]
If any of you come across this error: src/datasources/spotify-api.ts(3,31): error TS2307: Cannot find module '/src/models' or its corresponding type declarations. 
Then probably VSCode autocompleted the import to 'src/models' instead of '../models'.
Hope it helps someone!
